boards/arm/lpcxpresso55s36/doc/index.rst

```diff
@@ -77,6 +77,8 @@ already supported, which can also be re-used on this lpcxpresso55s36 board:
 +-----------+------------+-------------------------------------+
 | PWM       | on-chip    | pwm                                 |
 +-----------+------------+-------------------------------------+
+| CSS	    | on-chip	 | entropy			       |
++-----------+------------+-------------------------------------+
```

Comment:
This doesn't seem to align with the above line

Reply:
in what sense

Reply:
oh, I think there are tabs instead of spaces, so it might look wrong depending how you view it. I'll fix it